soknad/src/main/java/no/nav/k9/søknad/ytelse/psb/v1/PleiepengerSyktBarnValidator.java

```diff
@@ -172,30 +172,14 @@ private void validerTilsynsordning(Tilsynsordning tilsynsordning, Periode søkna
         if (tilsynsordning == null) {
             return;
         }
-        if (TilsynsordningSvar.JA == tilsynsordning.getiTilsynsordning())
-            if (tilsynsordning.getOpphold() == null || tilsynsordning.getOpphold().isEmpty()) {
+        if (tilsynsordning.getiTilsynsordning())
+            if (tilsynsordning.getPerioder() == null || tilsynsordning.getPerioder().isEmpty()) {
                 feil.add(new Feil("tilsynsordning.perioder", PÅKREVD, "Det må minst være en periode med opphold."));
```

Comment:
Trenger vi et eget flagg for "iTilsynsordning" hvis det er krav om minst én periode?

Reply:
👍